features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/members/RoomMemberListView.kt

```diff
@@ -135,6 +174,108 @@ fun RoomMemberListTopBar(
     )
 }
 
+@OptIn(ExperimentalMaterial3Api::class)
+@Composable
+private fun RoomMemberSearchBar(
+    query: String,
+    state: RoomMemberSearchResultState,
+    active: Boolean,
+    modifier: Modifier = Modifier,
+    placeHolderTitle: String = stringResource(io.element.android.libraries.ui.strings.R.string.common_search_for_someone),
+    onActiveChanged: (Boolean) -> Unit = {},
+    onTextChanged: (String) -> Unit = {},
+    onUserSelected: (RoomMember) -> Unit = {},
```

Comment:
I don't really have a strong opinion on this, but if a callback is required for the component to work properly I usually don't give it a default value.

Reply:
I use to add default lambda to callback in `Composable` as this is most of the time the case in Compose components and it helps to add `@Preview` annotation or to use the component without implementing the callback at the beginning but this is maybe more a personal preference than a rule and not really a good practice :)